iterations/iterations.r

```diff
@@ -0,0 +1,69 @@
+library(httr)
+library(jsonlite)
+library(ggplot2)
+library(DescTools)
+library(tidyverse)
+library(magrittr)
+library(rlang)
+library(lubridate)
+library(anytime)
+library(readr)
+library(yaml)
+
+#### 1: Beginning of script
+
+# Load function for posting GQL-queries and retrieving data: 
+source("functions/GQL_function.r")
+
+# The URL we will use is stored below: 
+
+configs <- 
+  read_yaml("vegvesen_configs.yml")
+
+
+gql_metadata_qry <- read_file("gql-queries/station_metadata.gql")
+
+# Let's try submitting the query: 
+
+stations_metadata <-
+  GQL(
+    query=gql_metadata_qry,
+    .url = configs$vegvesen_url
+    ) 
+
+
+#### 2: Transforming metadata
+
+source("functions/data_transformations.r")
```

Comment:
Remember to change this to the correct path, as you have changed the name of the script to "data_transformations_sol.r"